downloader/history_downloader/history_manager.py

```diff
@@ -0,0 +1,40 @@
+import ccxt
+import time
+
+
+class HistoryDownloader:
+
+    def __init__(self, exchange_name: str, crypto_symbol: str, days_back: int):

```

Comment:
`days_back` is somehow impractical don't you think? With that way we should specify eg: 745 days back... instead of just pass a starting date.

Reply:
Yeah, in some way it might be not comfortable, now the user is able to pass the date